trace-forward/trace-forward.cabal

```diff
@@ -0,0 +1,99 @@
+cabal-version:       2.4
+name:                trace-forward
+version:             0.1.0
+synopsis:            See README for more info
+description:         See README for more info
+license:             Apache-2.0
+license-file:        LICENSE
+copyright:           2021 Input Output (Hong Kong) Ltd.
+author:              IOHK
+maintainer:          operations@iohk.io
+build-type:          Simple
+extra-doc-files:     README.md
+                     CHANGELOG.md
+
+common common-options
+  build-depends:       base >=4.12 && <5
+
+  ghc-options:         -Wall
+                       -Wcompat
+                       -Widentities
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+  if impl(ghc >= 8.0)
+    ghc-options:       -Wredundant-constraints
+  if impl(ghc >= 8.2)
+    ghc-options:       -fhide-source-paths
+  if impl(ghc >= 8.4)
+    ghc-options:       -Wmissing-export-lists
+                       -Wpartial-fields
```

Comment:
There's a lot of IFs for versions of GHC that `cardano-node` doesn't support anymore?

What's the story here?

Reply:
Well, I just copied this section from another Haskell project (to be precise, from `iohk-monitoring-framework`).

Reply:
Should I remove it?

Reply:
Do we need to support `8.0`, `8.2` and `8.4`? On CI we only build with `8.10` right now.

I wouldn't use `-fhide-source-path`.

Reply:
@denisshevchenko, I would have used the `cardano-node` cabal definitions as baseline, instead of `iohk-monitoring-framework`.

Reply:
Done.